.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+	"presets": ["es2015", "es2016", "react"],
+	"plugins": ["babel-plugin-transform-class-properties"]
```

Comment:
Can we simply use the babel config within `react-scripts` or is there some reason why that wouldn't work?